src/constants/db.ts

```diff
@@ -0,0 +1,117 @@
+export const users = [
+  {
+    id: 1,
+    avatar: 'www.example.url/picture.png',
+    bgImage: 'www.example.url/picture.png',
```

Comment:
let's use this values for avatars and background images, put your urls here instead of having them as a constant